Makefile

```diff
@@ -232,3 +232,7 @@ _work/venv/.stamp: docs/requirements.txt
 	python3 -m venv ${@D}
 	. ${@D}/bin/activate && pip install wheel && pip install -r $<
 	touch $@
+
+helm:
+	cp -rf deployments/operator/* charts/operator
```

Comment:
@rmstoi did you try the `make generate` approach?

Reply:
```
diff --git a/Makefile b/Makefile
index 4b1f81b1..648d5455 100644
--- a/Makefile
+++ b/Makefile
@@ -95,6 +95,9 @@ generate:
        $(CONTROLLER_GEN) crd:crdVersions=v1 \
                paths="./pkg/apis/..." \
                output:crd:artifacts:config=deployments/operator/crd/bases
+       $(CONTROLLER_GEN) crd:crdVersions=v1 \
+               paths="./pkg/apis/..." \
+               output:crd:artifacts:config=charts/operator/crds
        $(CONTROLLER_GEN) crd:crdVersions=v1 \
                paths="./pkg/apis/fpga/..." \
                output:crd:artifacts:config=deployments/fpga_admissionwebhook/crd/bases
$ make generate
$ ls charts/operator/crds
```

Reply:
@mythi great idea with makefile change. I pushed updated PR.